Api/repository/entities/Expense.cs

```diff
@@ -0,0 +1,13 @@
+public class Expense : IEntityBase
```

Comment:
please review the [PR](https://github.com/programacion-6/ExpenseTrackerGrupo5/pull/5/files), because I changed the entities and I explained why

Reply:
Okay, I'll check it out.